python/xorbits/_mars/utils.py

```diff
@@ -1664,7 +1664,7 @@ def get_func_token(func):
 
 
 def _get_func_token_values(func):
-    if hasattr(func, "__code__"):
+    if hasattr(func, "__code__") and len(func.__code__.co_code) > 0:
```

Comment:
Could `co_code` be None?

Reply:
In latest python version, cython function's co_code is an empty string.